lib/validators/assignee.js

```diff
@@ -13,17 +13,19 @@ class Description extends Validator {
       'pull_request.milestoned',
       'pull_request.demilestoned',
       'pull_request.unlabeled',
+      'pull_request.assigned',
+      'pull_request.unassigned',
       'pull_request.synchronize',
       'issues.opened'
     ]
   }
 
   async validate (context, validationSettings) {
     let assignees = this.getPayload(context).assignees
-
+    console.log('assignee', assignees.map(assignee => assignee.login))
```

Comment:
Not a big deal but we may want to remove this console log.

Reply:
done